app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
You should specify inner type of list
```suggestion
def calculate_team_total_rating(team: list[Player]) -> int:
```

Reply:
do it